fair_mango/dataset/dataset.py

```diff
@@ -234,12 +508,81 @@ def get_real_target_for_one_group(self, sensitive: Sequence[str]) -> pd.DataFram
         return result.squeeze()
```

Comment:
did you create a task for that one?

Reply:
I removed `.squeeze()`. Now the function always returns `pandas.DataFrame`.